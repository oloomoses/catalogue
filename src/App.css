.stockCard {
  width: 250px;
  margin: 1rem;
}

.profit {
  color: grey;
  margin-right: 1rem;
}

.changes {
  margin-right: 5px;
}

.ticker {
  font-size: 2rem !important;
  font-weight: bolder !important;
}

.companyName {
  font-weight: bold !important;
  color: grey;
}
