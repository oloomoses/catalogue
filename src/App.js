import './App.css';
import Companies from './components/companies/Companies';

function App() {
  return (
    <div className="App">
      <Companies />
    </div>
  );
}

export default App;
