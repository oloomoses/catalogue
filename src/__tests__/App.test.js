import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import App from '../App';

describe('renders correctly', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<App />);
  });

  it('Contains the Navigation component', () => {
    expect(wrapper.find('Navigation')).toBeTruthy();
  });

  it('Contains the Companies component', () => {
    expect(wrapper.find('Companies')).toBeTruthy();
  });

  it('Contains the Sectors component', () => {
    expect(wrapper.find('Sectors')).toBeTruthy();
  });

  it('Contains the Company component', () => {
    expect(wrapper.find('Company')).toBeTruthy();
  });

  it('matches the snapshot', () => {
    const tree = renderer.create(<App />);
    expect(tree).toMatchSnapshot();
  });
});
