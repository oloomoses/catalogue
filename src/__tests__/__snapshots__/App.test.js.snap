// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly matches the snapshot 1`] = `
Array [
  <nav
    className="navbar navbar-expand navbar-dark bg-primary sticky-top"
  >
    <div
      className="container"
    >
      <a
        className="navbar-brand"
        href="/"
      >
        Stocks
      </a>
      <div
        className="mr-auto navbar-nav"
        onKeyDown={[Function]}
      >
        <a
          className="nav-link"
          data-rb-event-key="/"
          href="/"
          onClick={[Function]}
          onKeyDown={[Function]}
        >
          Home
        </a>
        <a
          className="nav-link"
          data-rb-event-key="/sectors"
          href="/sectors"
          onClick={[Function]}
          onKeyDown={[Function]}
        >
          Sectors Performance
        </a>
      </div>
      <form
        className="form-inline"
      >
        <select
          className="mr-sm-2 form-control"
          onChange={[Function]}
          placeholder="Search"
        >
          <option
            value="actives"
          >
            Most Active
          </option>
          <option
            value="losers"
          >
            Losers
          </option>
          <option
            value="gainers"
          >
            Gainers
          </option>
        </select>
        <button
          className="btn btn-outline-light"
          disabled={false}
          type="button"
        >
          Select Category
        </button>
      </form>
    </div>
  </nav>,
  <div
    className="container"
  >
    <div
      className="row"
    />
  </div>,
]
`;
